### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by anastasiiashchoholieva**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(team: list[Player]) -> None:
+    return sum([player.get_rating() for player in team])
```
Comment (ArsenPidhoretskyi) [inline app/main.py:7]: No need converting to list
```suggestion
    return sum(player.get_rating() for player in team)
```

File: app/players/player.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+    def __init__(self, nickname: str) -> None:
+        self.nickname = nickname
+
+    @abstractmethod
+    def get_rating(self) -> None:
```
Comment (ArsenPidhoretskyi) [inline app/players/player.py:9]: it returns int. You must specify type which will return all child classes

File: app/players/player.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+        pass
+
+    @abstractmethod
+    def player_info(self) -> None:
```
Comment (ArsenPidhoretskyi) [inline app/players/player.py:13]: It returns str

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

**Review by olena-hul — APPROVED** [review]

**Comment by olena-hul** [comment]
Good job!

**Pull request closed without merging**